src/components/booksitem.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './App.css';
+
+export default function booksitem({ list }) {
+  return (
+
+    <div>
+      <div className="book-oder">
+        <div className="info">
+          <span className="category">{list.category}</span>
+          <h3 className="title">{list.title}</h3>
+          <span className="author">{list.author}</span>
+        </div>
+        <div className="add-features">
+          <button type="button">Comment</button>
+          <div className="vert-line" />
+          <button type="button">Remove</button>
+          <div className="vert-line" />
+          <button type="button">Edit</button>
+        </div>
+      </div>
+      <div className="book-progress" />
+      <div className="progress-update" />
+    </div>
```

Comment:
- [ ] After everything, I think your code can look something like this for the various components :+1: 

```

// Book component
function Book({ title, author }) {
  return (
    <div>
      <h3>Title: {title}</h3>
      <p>Author: {author}</p>
      <button>Remove</button>
    </div>
  );
}

```


```
// BookList component
function BookList({ books }) {
  return (
    <div>
      {books.map((book) => (
        <Book key={book.id} title={book.title} author={book.author} />
      ))}
    </div>
  );
}

```
```
// Books component
export default function Books() {
  const [books] = useState([
    {
      id: 1,
      title: 'Romeo and Juliet',
      author: 'William Shakespeare',
      category: 'Non-fiction',
      completed: false,
    },
    {
      id: 2,
      title: 'Art of Hustling',
      author: 'Dj Sbu',
      category: 'Non-fiction',
      completed: false,
    },
  ]);

  return (
    <div>
      <BookList books={books} />
    </div>
  );
}

```